xds/src/main/java/io/grpc/xds/LoadReportClient.java

```diff
@@ -401,6 +387,28 @@ private void cleanUp() {
     }
   }
 
+  private static final class LoadStatsEntity {
+    private final LoadStatsStore loadStatsStore;
+    private final Stopwatch stopwatch;
+
+    private LoadStatsEntity(LoadStatsStore loadStatsStore, Stopwatch stopwatch) {
+      this.loadStatsStore = loadStatsStore;
+      this.stopwatch = stopwatch;
+      stopwatch.reset().start();
```

Comment:
Better to pass a StopwatchSupplier in constructor, and create a new stopwatch here, so that it's clear that the stopwatch's lifecycle is solely managed by the LoadStatsEntity.

Reply:
Updated to take the stopwatch supplier from the LrsClient directly.